README.md

```diff
@@ -45,8 +45,91 @@ Installation will vary depending on whether you use the `yarn` or `npm` packages
 The steps are simple:
 
 1. Clone this repository
-2. [Install `yarn` on your system using these instructions](https://classic.yarnpkg.com/en/docs/install)
-3. Follow these steps next
+2. Install Yarn on your system following these instructions
+
+### Install Yarn on Windows Using NPM
+
+- [Download the Windows installer](https://nodejs.org/en/download/)
+- After choosing the path, double-click to install. Then give access to run the application
+- Install Yarn by running the following command
+
+```terminal
+$ npm install --global yarn
+```
+
+- Check Yarn installation
+
+```terminal
+$ yarn -version
+```
+
+### Install Yarn on Windows Using msi File
+
+- [Download the Windows installer](https://classic.yarnpkg.com/en/docs/install#windows-stable)
+- After choosing the path, double-click to install. Then give access to run the application
+
+- Check Yarn installation
+
+```terminal
+$ yarn -version
+```
+
+### Install Yarn on macOS Using NPM
+
+- [Click on the macOS Installer option to download the .pkg installer](https://nodejs.org/en/download/)
+- Run the Node.js installer
+- Verify Node.js Installation by running the following command in your terminal
```

Comment:
Use numbers, not bullets. People can reference point `#4` which is easier to find.